data_structures/stacks/infix_to_prefix_conversion.py

```diff
@@ -134,24 +166,20 @@ def infix_2_prefix(infix):
         ...
     IndexError: list index out of range
 
-    >>> infix_2_prefix('())')  # doctest: +NORMALIZE_WHITESPACE
-     Symbol  |  Stack  | Postfix
-    ----------------------------
-       (     | (       |
-       (     | ((      |
-       )     | (       |
-             |         | (
-    '('
+    >>> infix_2_prefix('())')
+    Traceback (most recent call last):
+        ...
+    ValueError: invalid expression
     """
-    infix = list(infix[::-1])  # reverse the infix equation
+    new_infix = list(infix[::-1])  # reverse the infix equation
```

Comment:
Would `reversed_infix` be a more self-documenting name to help the reader understand?

Reply:
Hmm yea that makes more sense